### PR TITLE
LUI-77 | Preethi, Chethan | Feedback on Concept attributes

### PR DESCRIPTION
**Pull request by preethi29**

### CODE REVIEW

File: omod/src/main/webapp/admin/concepts/conceptAttributeType.jsp [hunk 1]
```diff
@@ -7,7 +7,8 @@
 
 <script type="text/javascript">
     function confirmPurge() {
-        if (confirm("Are you sure you want to purge this object? It will be permanently removed from the system.")) {
+        <openmrs:message var="confirmMessage" code="ConceptAttributeType.confirm.purge"/>
```
Comment (dkayiwa) [inline omod/src/main/webapp/admin/concepts/conceptAttributeType.jsp]: @preethi29 are you sure this works? Did you test it out? :)

Comment (preethi29) [inline omod/src/main/webapp/admin/concepts/conceptAttributeType.jsp]: yes we tested this out and it worked

Comment (dkayiwa) [inline omod/src/main/webapp/admin/concepts/conceptAttributeType.jsp]: @preethi29 can you paste the context of the view source option for the running jsp?

Comment (preethi29) [inline omod/src/main/webapp/admin/concepts/conceptAttributeType.jsp]: How do i do that?

Comment (psbrandt) [inline omod/src/main/webapp/admin/concepts/conceptAttributeType.jsp]: Right-click > view source in the browser?

Comment (preethi29) [inline omod/src/main/webapp/admin/concepts/conceptAttributeType.jsp]: http://pastebin.com/u5R6GhPm

**Pull request merged**